datahub-web-react/.env

```diff
@@ -1,4 +1,5 @@
 PUBLIC_URL=/assets
 REACT_APP_THEME_CONFIG=theme_light.config.json
 SKIP_PREFLIGHT_CHECK=true
-BUILD_PATH=build/yarn
\ No newline at end of file
+BUILD_PATH=build/yarn
+MANAGE_POSTS_ENABLED=true
```

Comment:
what we've been trying to do is to create feature flags that are more global and can be set with env variables in our helm charts. this involves setting it in `application.yaml` with a default (true in this case) and passing that feature flag to the frontend in our app config graphql endpoint. An example PR doing exactly this and disabling a piece of the UI based on the flag is here: https://github.com/datahub-project/datahub/pull/8067

Reply:
Hmmm Thanks @chriscollins3456 !!